.github/styles/README.md

```diff
@@ -0,0 +1,27 @@
+# Style Guide
+The team follows the Google developer's and the Microsoft's documentation style guide. The style rules are configured with the help of Vale linter.
+
+## Prerequisites
+Please ensure that the below prerequisites are met before you configure vale linter. You should have:
+
+* [Brew](https://brew.sh) installed on your Mac, or [Chocolatey](https://chocolatey.org/) installed on your Windows machine.
+> For windows, you could also use the command prompt and curl command to install brew - `curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh`
```

Comment:
This is a part of the above bullet point right? Aligning it with three spaces will make it look much clearer. Similarly for the other points. 😃

```suggestion
   > For windows, you could also use the command prompt and curl command to install brew - `curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh`
```